backend/inspirehep/records/tasks.py

```diff
@@ -205,3 +207,28 @@ def remove_bai_from_literature_authors(uuids):
                 recid=record["control_number"],
                 uuid=record.id,
             )
+
+
+@shared_task(
+    acks_late=True,
+    retry_backoff=True,
+    autoretry_for=DB_TASK_EXCEPTIONS,
```

Comment:
the task will restart on `StaleDataError` and we don't have to IMO

Reply:
but I would retry for other db exceptions, maybe we can raise a custom one instead of stale data